packages/react/src/styles.scss

```diff
@@ -2,6 +2,12 @@
 // 🌍 Global
 //-------------------------
 
+// Enable math module to handle dividing operations due to deprecation of " / " operator
+// Reference https://sass-lang.com/documentation/breaking-changes/slash-div
+/* stylelint-disable */
+@use "sass:math";
+/* stylelint-enable */
+
```

Comment:
In the downstream Maximo monitor component, this is causing the following error. 

<img width="638" alt="image" src="https://user-images.githubusercontent.com/18572477/193832951-e0bd97fd-9010-4a83-992f-88caa8b159f4.png">


I know we have to re-arrange the scss imports to over come the issue. but It may create CSS style related regressions.